### DISCUSSION TITLE
Halo2能把证明丢给prover实现nova那样的过程吗

### DISCUSSION THREAD

**Discussion by Pupil1999** [z2o-k7e/halo2-step-by-step#31] [Q&A]
如题，想问问上一轮生成的proof内容能不能丢给下一轮中的电路？如果有的话，可以给个用例吗？

**Comment by Demian101** [comment]
@dajuguan Po : Nova 用的 Spartan，前后端没有解耦，目前还没有实现
@周洋：lookup —— Nova 还没有实现
Nova 前端用的 R1CS
